pkg/kubelet/types/pod_update.go

```diff
@@ -141,7 +141,7 @@ func (sp SyncPodType) String() string {
 }
 
 // IsCriticalPod returns true if the pod bears the critical pod annotation key or if pod's priority is greater than
-// or equal to SystemCriticalPriority. Both the default scheduler and the kubelet use this function
+// or equal to SystemCriticalPriority. Both the rescheduler(deprecated in 1.10) and the kubelet use this function
```

Comment:
Comment should be fixed since rescheduler is no longer deprecated.

Reply:
So, I think the statement is still valid just that we will delete rescheduler in next release. This is a stop gap solution.

Reply:
What's the long term fix (and if we rely on a deprecated feature, is it really deprecated?)

Reply:
Please find my comments below. While I am not familiar with deprecation policies and this may not sound like exact reason, last release had rescheduler deprecated too but we relied on it for DS pods.